src/components/Dashboard/FilingHistoryList/CompletedDissolution.vue

```diff
@@ -5,18 +5,18 @@
     <p v-if="isFirm">
       The statement of dissolution for {{entityTitle}} {{getEntityName || ''}} was successfully
       submitted on <strong>{{dissolutionDateSubmitted}}</strong>
-      with dissolution date of <strong>{{dissolutionDateTime}}</strong>.
+      with dissolution date of <strong>{{dissolutionDate}}</strong>.
       The {{entityTitle}} has been struck from the register and dissolved,
       and ceased to be a registered {{ entityTitle }}
-      under the {{actTitle}}.
+      under the {{actTitle}} Act.
     </p>
 
     <p v-if="!isFirm">
       The {{entityTitle}} {{getEntityName || ''}} was successfully
       <strong>dissolved on {{dissolutionDateTime}}</strong>.
       The {{entityTitle}} has been struck from the register and dissolved,
       and ceased to be an incorporated {{entityTitle.toLowerCase()}}
-      under the {{actTitle}}.
+      under the {{actTitle}} Act.
```

Comment:
Is this change for all type of company?

Reply:
Using the same act titles from the ConfirmDissolution file